box.007

```diff
@@ -1,42 +1,42 @@
 my text = [
-    "This is random text",
+    "This is a random text",
     "that will be boxed with this function.",
     "I have set it to be 3"
 ];
 
 boxify(text, 4);
 
-func boxify(strings_to_box, w_on_sides) {
+func boxify(stringsToBox, wOnSides) {
 
```

Comment:
This empty line at the start of a function isn't in line with code style either. (I must remember to mention that when I write them up.) There's another one at L73.

Reply:
Still an empty line here. L10 should be removed, preferably.

Reply:
```suggestion
- 
```